modules/validations/outdated_tags.js

```diff
@@ -116,6 +117,19 @@ export function validationOutdatedTags() {
             }
           })
         ];
+
+        if (nsiResult && nsiResult.matched) {
+          fixes.push(
+            new validationIssueFix({
+              title: t.html('issues.fix.tag_as_not.title', { name: nsiResult.matched.displayName }),
```

Comment:
The screenshot shows a QID in this spot, but that would be inscrutable to ordinary mappers, especially since there’s no link or even a way to select the text. On the other hand, if the correct and incorrect brands happen to share the same name, “Tag as **not** Noah’s Bagels” wouldn’t look like the right option either. How about “Tag as a different Noah’s Bagels”?

Reply:
> “Tag as not the same Noah’s Bagels”
cool, yes this is much clearer - I'll go with this

Reply:
<img width="350" alt="Screen Shot 2021-08-06 at 5 12 32 PM" src="https://user-images.githubusercontent.com/38784/128571658-154f69c3-13b0-4e0b-8b84-d3f0f234fa4e.png">